cpp/src/slice2java/GenCompat.cpp

```diff
@@ -210,6 +210,28 @@ writeParamList(Output& out, vector<string> params, bool end = true, bool newLine
     }
 }
 
+// Returns the "length" of the constructor parameter list from the given data member list.
+// All types are counted as 1, except for long and double which are counted as 2.
```

Comment:
Just to tie into the terminology used by the JVM.
```suggestion
// All types are counted as 1 unit, except for long and double which are counted as 2 units.
```